kubernetes/linux/main.sh

```diff
@@ -932,11 +932,15 @@ fi
 
 #start telegraf
 if [ "${MUTE_PROM_SIDECAR}" != "true" ]; then
-      /opt/telegraf --config $telegrafConfFile &
-      echo "telegraf version: $(/opt/telegraf --version)"
-      dpkg -l | grep fluent-bit | awk '{print $2 " " $3}'
+    if [ "${CONTROLLER_TYPE}" == "ReplicaSet" ] && [ "${SIDECAR_SCRAPING_ENABLED}" == "true" ]; then
```

Comment:
Can we use the same condition as livenessprobe? if [ "${CONTROLLER_TYPE}" == "ReplicaSet" ] && [ ! -z "${SIDECAR_SCRAPING_ENABLED}" ] && [ "${SIDECAR_SCRAPING_ENABLED}" == "true" ]; then

Reply:
Yeah true. May be we can remove from Livenessprobe as well then? Was just thinking to keep it the same at both places unless you think otherwise.

Reply:
This change doesnt consider cluster prometheus config. Telegraf only need to turned off if there is not cluster config enabled.